packages/bytecode-utils/README.md

```diff
@@ -10,24 +10,59 @@ yarn add @ethereum-sourcify/bytecode-utils
 
 ## Usage
 
+### Solidity Contracts
+
 ```ts
 import { decode } from "@ethereum-sourcify/bytecode-utils";
 
 const bytecodeRaw = "0x60806040526004361061003f5760003560e01...7265206c656e677468a2646970667358221220dceca8706b29e917dacf25fceef95acac8d90d765ac926663ce4096195952b6164736f6c634300060b0033"
 
-decode(bytecodeRaw);
+// For Solidity contracts
+decode(bytecodeRaw, AuxdataStyle.SOLIDITY);
 ```
 
-### Result
+**Result example**
 
 ```json
 {
-  "cbor": {
-    "bytes": "0xa2646970667358221220dceca8706b29e917dacf25fceef95acac8d90d765ac926663ce4096195952b6164736f6c634300060b",
-    "length": 51
-  },
   "ipfs": "QmdD3hpMj6mEFVy9DP4QqjHaoeYbhKsYvApX1YZNfjTVWp",
-  "solcVersion": "0.6.11"
-  "experimental": true
+  "solcVersion": "0.6.11",
+  "experimental": true,
+  "bzzr0": "...",
+  "bzzr1": "..."
+}
+```
+
+### Vyper Contracts
+
+```ts
+import { decode, AuxdataStyle } from "@ethereum-sourcify/bytecode-utils";
+
+const vyperBytecodeRaw = "0x..."; // Your Vyper contract bytecode
+
+// For Vyper contracts
+decode(vyperBytecodeRaw, AuxdataStyle.VYPER);
+```
+
+**Result example**
+
+```json
+{
+  "integrity": "...",
+  "runtimeSize": 1234,
+  "dataSizes": [32, 64, 128],
+  "immutableSize": 256,
+  "compiler": {
```

Comment:
I see in the test of your other PR that compiler looks like this: `compiler: '0.4.1'` (not an object). 

In the `VyperDecodedObject` it is defined like this:
```
compiler: {
    vyper: [number, number, number];
  };
  ```
  
  Not sure which of these three versions is right. This might also need a change in your other PR.